### PR TITLE
Add e2e tests with screenshot taking

### PR DESCRIPTION
**Pull request by davidyuk**
These tests are trying to open most of the pages of the Base app. Also, they are making screenshots of opened pages. I propose to store these screenshots in our repository to be able to compare visual changes made in PRs. Supposed that developer should run `npm run e2e` on the local machine and commit new screenshots.
Later should be possible to implement comparison of existing screenshots with new ones made by Travis while testing to check if committed screenshots are actual.

### CODE REVIEW

**Comment by dadaphl** [comment]
That is a very good idea. I think we should not depend on manual creation of those screenshots though. Also I don't like to store the screenshots in the repository. How about we create and deploy them to a webserver on push via travis?

**Comment by dadaphl** [comment]
Ok, so I think that you agree. Then let's remove the images before merging please.

**Comment by dadaphl** [comment]
i think travis-ci doesn't give us enough resources to build the screens

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+  chrome: stable
 language: node_js
 node_js:
 - '8'
 install:
 - npm install
 script:
-- npm run lint
-- npm run unit
+- npm test
```
Comment (dadaphl) [inline .travis.yml]: this doesn't work

**Comment by dadaphl** [comment]
@davidyuk please provide an update on this PR. Are you going to continue working on it?

**Comment by davidyuk** [comment]
@dadaphl I still have some weird errors in Travis environment, I'm going to investigate this later. At least, I can run e2e tests in Jenkins that is probably more predictable.

**Comment by davidyuk** [comment]
I'm closing this PR because screenshot taking using Nightwatch is not needed, as far as we are using more user-friendly Cypress e2e test runner. But in general screenshot taking still can make testing easier.

**Pull request closed without merging**